test/fancylogger.py

```diff
@@ -369,6 +369,46 @@ def test_normal_logging(self):
         fancylogger.logToScreen(enable=False, handler=handler)
         sys.stderr = _stderr
 
+    def test_fancyrecord(self):
+        """
+        Test fancyrecord usage
+        """
+        logger=fancylogger.getLogger()
+        self.assertTrue(logger.fancyrecord == True)
```

Comment:
use `assertEqual` when comparing values, which will lead to clear test failure messages